### PR TITLE
VDS-2329: Add new member

### PR DESCRIPTION
**Pull request by asvishnyakov**
Description: 

--

QA-test: VDS-2331

Demo-test: VDS-2332

Download artifact URL: ghcr.io/virtocommerce/demo-xapi-app:1.0.0-pr-61-vds-2329-8077ba74

### CODE REVIEW

File: src/app/modules/members/components/add-member/add-member.component.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    firstName: new FormControl(''),
+    lastName: new FormControl(''),
+    gender: new FormControl(''),
+    email: new FormControl(''),
+    userName: new FormControl(''),
+    password: new FormControl(''),
+    confirmPassword: new FormControl(''),
+  });
+
+  genderDictionaryItems = this.store.select(selectGenderDictionaryItems);
```
Comment (ilyawzrd) [inline src/app/modules/members/components/add-member/add-member.component.ts]: I think you forgot a $ at the end - it's an observable

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

File: src/app/modules/members/components/add-member/add-member.component.ts (hunk 1 above)
Comment (asvishnyakov) [reply src/app/modules/members/components/add-member/add-member.component.ts]: Fixed

**Review by ilyawzrd — APPROVED** [review]

File: src/app/modules/members/store/members.actions.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { ApolloError } from '@apollo/client/errors';
+import { createAction, props } from '@ngrx/store';
+import { getDictionaryDynamicProperty } from 'src/app/graphql/types/getDictionaryDynamicProperty';
+import { Member } from 'src/app/models/member.model';
+
+export const getGender = createAction(
```
Comment (trueboroda) [inline src/app/modules/members/store/members.actions.ts:6]: Rename to getGenderValues or Items

**Review by trueboroda — APPROVED** [review]

File: src/app/modules/members/store/members.actions.ts (hunk 2 above)
Comment (asvishnyakov) [reply src/app/modules/members/store/members.actions.ts:6]: Already passed test on QA, so will rename to getGenderDictionaryItems in next PR

**Pull request merged**